lib/gds_api/email_alert_api.rb

```diff
@@ -220,15 +220,21 @@ def change_subscription(id:, frequency:)
     )
   end
 
-  # Create an authentication token for a subscriber
+  # Deprecated method, issues warning and runs alias send_subscriber_verification_email
+  def create_auth_token(address:, destination:, redirect: nil)
```

Comment:
Could lose this, wasn't sure if giving a warning was better than just aliasing create_auth_token to send_subscriber_verification_email